examples/python/phi-3-tutorial.md

```diff
@@ -75,7 +75,7 @@ The `-l` argument is the length of output you would like to generate with the mo
 
 ```bash
 curl https://raw.githubusercontent.com/microsoft/onnxruntime-genai/main/examples/python/phi3-qa.py -o phi3-qa.py
-python phi3-qa.py -m Phi-3-mini-128k-instruct-onnx/directml/directml-int4-awq-block-128 -l 2048
+python phi3-qa.py -m {modelPath}  -l 2048
```

Comment:
You'll want to change the text above too - to refer to the variable rather than the DML model